distest/interface.py

```diff
@@ -152,6 +152,86 @@ def checkReaction(reaction, user):
         await self.channel.send(content)
         return await self.wait_for_message()
 
+    async def assert_embed_equals(
+        self,
+        message: discord.Message,
+        matches: discord.Embed,
+        attributes_to_prove: list = None,
+    ):
+        """
+        If ``matches`` doesn't match the embed of ``message``, fail the test.
+        :param message: original message
+        :param matches: embed object to compare to
+        :param attributes_to_prove: a string list with the attributes of the embed, which are to compare
```

Comment:
Looks good, just plz explain `attributes_to_prove` a bit better

Reply:
Should be done.